lib/election.js

```diff
@@ -155,9 +157,11 @@ api.getBlockElectors = (ledgerNode, blockHeight, callback) => {
             return aPreviousElector - bPreviousElector;
           }
 
+          // FIXME: does this comment need to be updated?  The existing
+          // implementation does not involve the manifest.
```

Comment:
We do need to involve the manifest or some value from the previous block that changes with every new block and can't be controlled by any particular party. We just want to avoid people creating voter IDs that hash to a low hash in order to give themselves preference.

Reply:
@dlongley are you saying the comment `3. sort by hex hash(manifest + voter.id)` is accurate, and can be left as is?  And the FIXME at L160 just needs to be removed?

or are you saying we need to alter the inputs to the hash?

Reply:
I'm saying comment we need to alter the inputs to the hash. Either we need to match what comment `3` is saying (if that's easy) or we need to use another input (if that's hard/makes no sense).